FinniversKit.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -18,7 +17,16 @@
         "revision" : "b44b563e6ddc5a613a107fb57626047a7f1d4e1a",
         "version" : "1.9.666"
       }
+    },
```

Comment:
reverted package.resolved to the previous commit, since this build failed:
https://github.com/finn-no/FinniversKit/commit/7beb04072878004cf1913a56cbcf8c4bc43f810d

Reply:
Since we now use Xcode 15.3 we should be fine with the current setup, and I guess it'll be updated by either of us during the next PR 🙂